src/main/java/com/technocredits/orangeHRM/pages/LoginPage.java

```diff
@@ -1,21 +1,33 @@
 package com.technocredits.orangeHRM.pages;
 
-public class LoginPage {
-
-	private static LoginPage loginPage;
-
-	private LoginPage() {
-
+import java.io.IOException;
+import java.util.Properties;
+
+import com.technocredits.orangeHRM.base.PredefinedMethods;
+import com.technocredits.orangeHRM.util.PropertyFileReader;
+
+public class LoginPage extends PredefinedMethods{
+	private static LoginPage loginPage = null;
```

Comment:
What is the need to assign this variable null ?
```suggestion
	private static LoginPage loginPage = null;
```

Reply:
No need to assign null value to static variable. Updated same in code.